### PR TITLE
Adds circle.yml file for continuous integration

### PR DESCRIPTION
**Pull request by ncrmro**
Build and runs the stack then simply curls the server to make sure it's live and sending data.

@ivarayut you'll need to activate this repo on circle ci for this to take affect.

You can see it build my fork with this commit [here](https://circleci.com/gh/ncrmro/relay-fullstack/14)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          echo "Download and install Yarn."
+          curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
+        else
+          echo "The correct version of Yarn is already installed."
+        fi
+  override:
+      - yarn
+      - webpack --config webpack.config.js
+      - npm run update
+      - npm run build-server
```
Comment (Neitsch) [inline circle.yml]: Either choose yarn or npm.

**Review by Neitsch — CHANGES_REQUESTED** [review]

File: circle.yml (hunk 1 above)
Comment (ncrmro) [reply circle.yml]: Yarn is used to install dependencies and npm is just piggybacking on package.json commands.

Yarn is much faster to install dependencies, wouldn't you agree?

Comment (Neitsch) [reply circle.yml]: Yes, absolutely, but why use npm at all? Just use yarn run

**Review by Neitsch — APPROVED** [review]

File: circle.yml (hunk 1 above)
Comment (ncrmro) [reply circle.yml]: Still not sure of everything yarn can do, switched commands 👍

**Comment by Neitsch** [comment]
@lvarayut will you enable CI?

**Comment by lvarayut** [comment]
Yes, I will enable it very soon.

**Comment by lvarayut** [comment]
@ncrmro, @Neitsch, I normally use Travis as my CI, which seems to be very fast and easy to use. Just wanted to know your preference before merging this PR. Do you also prefer Travis or you still want to go with CircleCI?

**Comment by Neitsch** [comment]
I'm also a Travis guy, but why not try Circle CI out? ;)

**Comment by ncrmro** [comment]
@lvarayut  @Neitsch circle ci is is also quite fast they have 4gb ram builds. I've used them pretty extensively. Not much experience with Travis myself

**Comment by lvarayut** [comment]
Alright. Let's try that out 👍

**Comment by ncrmro** [comment]
@lvarayut @Neitsch Whooo!

**Pull request merged**